src/main/java/io/rocketpartners/cloud/action/sql/SqlDb.java

```diff
@@ -254,27 +256,29 @@ protected void shutdown0()
    @Override
    public String upsert(Table table, Map<String, Object> row) throws Exception
    {
-
       if (isType("h2"))
       {
          return h2Upsert(table, row);
       }
-      //    else if (isType("mysql"))
-      //    {
-      //       return mysqlUpsert(table, row);
-      //    }
-      //    
+      else if (isType("mysql"))
+      {
+         return StringUtils.join(mysqlUpsert(table, row), ',');
```

Comment:
@wellsb1 Since this method's return type is **String**,  here List of keys are returning as comma separated.